### PR TITLE
LW-R8: Empty Author Query appears for the author user (LEAN-2001)

### PR DESCRIPTION
**Pull request by asouqi**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@manuscripts/transform",
   "description": "ProseMirror transformer for Manuscripts applications",
-  "version": "1.0.1",
+  "version": "1.0.1-LEAN-2001",
```
Comment (hhusban) [inline package.json:4]: shouldn't this be changed to a regular versioning

Comment (asouqi) [reply package.json:4]: so, there is no need for testing that first. before merging it.
CC: @mnatsheh

**Comment by mnatsheh** [comment]
@asouqi @hhusban what's the status of this? Sorry I missed the previous comment.

**Comment by asouqi** [comment]
> what's the status of this? Sorry I missed the previous comment.

oh nevermind I will close this PR. as we fix this issue on the comment rework [PR](https://github.com/Atypon-OpenSource/manuscripts-transform/pull/4)

**Pull request closed without merging**